### PR TITLE
ui: journal embedded search view and number of papers

### PR DESCRIPTION
**Pull request by karolina-siemieniuk-morawska**
* ref: cern-sis/issues-inspire#71

### CODE REVIEW

File: ui/src/journals/containers/DetailPageContainer.tsx
Comment (drjova) [inline ui/src/journals/containers/DetailPageContainer.tsx:118]: why do we need `toString`?

File: ui/src/journals/containers/DetailPageContainer.tsx
Comment (drjova) [inline ui/src/journals/containers/DetailPageContainer.tsx:118]:
```suggestion
            <JournalPapers journalName={shortTitle} />
```

File: ui/src/journals/components/JournalPapers.tsx
Comment (MJedr) [inline ui/src/journals/components/JournalPapers.tsx:18]: is it the same style of info as in other embedded searches?

File: ui/src/journals/components/JournalPapers.tsx
Comment (MJedr) [inline ui/src/journals/components/JournalPapers.tsx:9]: I don't think that it's a correct field, it should be without `raw`

File: ui/src/journals/containers/DetailPageContainer.tsx
Comment (MJedr) [inline ui/src/journals/containers/DetailPageContainer.tsx:38]: journal title and short title are required by schema

**Review by MJedr — COMMENTED** [review]
please do not merge until backend is merged

File: ui/src/journals/components/JournalPapers.tsx
Comment (karolina-siemieniuk-morawska) [reply ui/src/journals/components/JournalPapers.tsx:18]: It's the same as experiments, institution and conference embedded search results

File: ui/src/journals/components/JournalItem.tsx
Comment (drjova) [inline ui/src/journals/components/JournalItem.tsx:61]: `null`

File: ui/src/journals/containers/DetailPageContainer.tsx
Comment (drjova) [inline ui/src/journals/containers/DetailPageContainer.tsx:103]: why we change it?

File: ui/src/journals/containers/DetailPageContainer.tsx
Comment (drjova) [inline ui/src/journals/containers/DetailPageContainer.tsx:76]: are we sure it won't fail with publisher `undefined`?

File: ui/src/journals/containers/DetailPageContainer.tsx
Comment (karolina-siemieniuk-morawska) [reply ui/src/journals/containers/DetailPageContainer.tsx:103]: @MJedr noticed [above](https://github.com/inspirehep/inspirehep/pull/2697#discussion_r999190392) that correct query should be without raw (it works with and without, but since I changed it in JournalPapers.tsx I changed here as well to be consistent)

File: ui/src/journals/containers/DetailPageContainer.tsx
Comment (karolina-siemieniuk-morawska) [reply ui/src/journals/containers/DetailPageContainer.tsx:76]: This `publisher &&` prevents from executing if undefined the same as ternary operator `publisher ?` (checked and if publisher is undefined it just does not render subTitle)

File: ui/src/journals/containers/SearchPageContainer.tsx
Comment (drjova) [inline ui/src/journals/containers/SearchPageContainer.tsx:23]: is a string or a number? We are using it inconsistently for example [here](https://github.com/inspirehep/inspirehep/pull/2697/files#diff-b28d9d340beff2217c916f019c8f197385f9532ff78264067457d37b0ec7bc5bR31) we are expecting it as a number

Comment (drjova) [reply ui/src/journals/containers/SearchPageContainer.tsx:23]: @karolina-siemieniuk-morawska ☝️

Comment (karolina-siemieniuk-morawska) [reply ui/src/journals/containers/SearchPageContainer.tsx:23]: Unified to type number (correct type) across files

**Review by drjova — APPROVED** [review]

**Pull request merged**
